Bottom: 3f4bb892a5a463c5e1aaa33760b1399433180122
Top:    20da1c7373ce22b26b293db83f3a460fb9dedc35
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-07-05 17:47:21 +0100

Refresh of move-arc-from-included-angle

---

diff --git a/src/misc.c b/src/misc.c
index 59d3336..83daec5 100644
--- a/src/misc.c
+++ b/src/misc.c
@@ -367,12 +367,14 @@ calc_arc_from_points_and_included_angle (PointType *p1, PointType *p2, Angle inc
   *cy = hy + unit_hcy * c_to_h_dist;
 
   if (radius != NULL)
-    {
-      *radius = sqrt ((p1->X - *cx) * (p1->X - *cx) +
-                      (p1->Y - *cy) * (p1->Y - *cy));
-    }
+    *radius = sqrt ((p1->X - *cx) * (p1->X - *cx) +
+                    (p1->Y - *cy) * (p1->Y - *cy));
+
+  if (start_angle != NULL)
+    *start_angle = TO_DEGREES (atan2 ((p1->Y - v->cy), -(p1->X - v->cx)));
 
-#warning start_angle and delta_angle are not computed yet!
+  if (delta_angle != NULL)
+    *delta_angle = included_angle;
 }
 
 /* ---------------------------------------------------------------------------
